retriever/lib/engine.py

```diff
@@ -683,8 +683,8 @@ def table_name(self, name=None, dbname=None):
     def to_csv(self):
         # Due to Cyclic imports we can not move this import to the top
         from retriever.lib.engine_tools import sort_csv
-        for _ in list(self.script.urls.keys()):
-            table_name = self.table_name()
+        for table in list(self.script.tables.keys()):
```

Comment:
let us change the table name because we have table as a global. 
table_i or table_n or whatever you are fine with